dvc/repo/logs.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import os
+from typing import List
+
+from dvc.repo import Repo
+from dvc.visualization import embed, metrics_embedding, plots_embeddings
+
+logger = logging.getLogger(__name__)
+
+
+class Logs:
```

Comment:
Naming `dvclive`/`logs` is not set in stone. It will probably be changed. Ideas are welcomed.